src/ast/implementation/meta/using_for_directive.ts

```diff
@@ -5,31 +5,54 @@ import { IdentifierPath } from "./identifier_path";
 
 export class UsingForDirective extends ASTNode {
     /**
-     * A library type
+     * A library type name or identifier.
+     * One of vLibraryName or vFunctionList should always be set.
      */
-    vLibraryName: UserDefinedTypeName | IdentifierPath;
+    vLibraryName?: UserDefinedTypeName | IdentifierPath;
+
+    /**
+     * Function list for file-level using-for directives.
+     * One of vLibraryName or vFunctionList should always be set.
+     */
+    vFunctionList?: IdentifierPath[];
```

Comment:
So the `functionList` property in the raw AST is *always* an `IdentifierPath` ? Even when single identifier are passed in?

Reply:
Ok, verified that its always IdentifierPath.